package-lock.json

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
Do not checking package-lock.json as its not related to this change.

Reply:
It's a whitespace only change (line endings probably). There is no change that would impact anything.

Reply:
This file wasnt added to git repo earlier and is being added as part of your change.

Reply:
Then it was correct to add it (https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5).